action.yaml

```diff
@@ -62,7 +62,7 @@ runs:
 
         $vsixXml.Save("${{ inputs.manifest-file }}") | Out-Null
 
-        Write-Output "version=$version" >> $GITHUB_OUTPUT
+        Write-Output "version=$version" >> $Env:GITHUB_OUTPUT
```

Comment:
Ok this works for `windows-latest` but fails for `ubuntu-latest` -- will need to ensure BOTH work here.

Reply:
Yes, see: https://github.com/timheuer/runner-testing/actions/runs/3461907153/jobs/5780108398#step:5:1 -- I thought it weird as well that powershell would act differently.

Reply:
Actually found my issue wasn't specifying shell so was defaulting to bash.  Ok, this is good...will merge.